lib/db.js

```diff
@@ -721,6 +758,16 @@ module.exports = function (
     return this.pool.post('/account/' + account.uid.toString('hex') + '/verifyEmail')
   }
 
+  DB.prototype.verifyToken = function (tokenVerificationId, token) {
```

Comment:
API nit: passing the `token` here makes it look like this will verify that specific token, but the implementation actually verifies *all* tokens that use the given `tokenVerificationId`.  I wonder if we can make it more obvious that we're only passing in the `uid`.